v3/internal/templates/_common/Taskfile.tmpl.yml

```diff
@@ -310,17 +311,56 @@ tasks:
       DESKTOP_FILE: '{{ "{{.APP_NAME}}" }}.desktop'
       OUTPUT_DIR: '../../bin'
 
+  create:rpm:
+    summary: Creates a RPM package
+    dir: build/rpm
+    platforms: [ linux ]
+    deps:
+      - task: build:linux
+        vars:
+          PRODUCTION: "true"
+      - task: generate:linux:dotdesktop
+    cmds:
+      # Copy binary + icon to rpmbuild dir
+      - cp {{ "{{.APP_BINARY}}" }} {{ "{{.APP_NAME}}" }}
+      - cp ../appicon.png '{{ "{{.APP_NAME}}" }}'.png
+      - cp ../rpm/app.spec '{{ "{{.APP_NAME}}" }}'.spec
+      - sed -i -e 's/@NAME@/{{ "{{.APP_NAME}}" }}/g'
+              -e 's/@SUMMARY@/{{ "{{.APP_SUMMARY}}" }}/g'
+              -e 's/@VERSION@/{{ "{{.APP_VERSION}}" }}/g'
+              -e 's/@DESCRIPTION@/{{ "{{.APP_DESC}}" }}/g'
+              -e 's/@AUTHOR@/{{ "{{.APP_AUTHOR}}" }}/g'
+              -e 's/@AUTHOR_EMAIL@/{{ "{{.AUTHOR_EMAIL}}" }}/g'
+              {{ "{{.APP_NAME}}" }}.spec
+      # Generate RPM
+      - wails3 generate rpm -binary {{ "{{.APP_NAME}}" }} -icon {{ "{{.ICON}}" }} -desktopfile {{ "{{.DESKTOP_FILE}}" }} -outputdir {{ "{{.OUTPUT_DIR}}" }} -builddir {{ "{{.ROOT_DIR}}" }}/build/rpm
+      - mv {{ "{{.ROOT_DIR}}" }}/build/rpm/$(uname -m)/*.rpm {{ "{{.OUTPUT_DIR}}" }}
+    vars:
+      APP_NAME: '{{ "{{.APP_NAME}}" }}'
+      APP_SUMMARY: 'Hello Wails'
+      APP_VERSION: '1.0.0'
+      APP_DESC: 'Hello Wails is demo application developed on Wails.'
+      APP_URL: 'https://github.com/wailsapp/wails'
+      APP_AUTHOR: "Lea Anthony"
+      AUTHOR_EMAIL: "lea.anthony@gmail.com"
+      APP_BINARY: '../../bin/{{ "{{.APP_NAME}}" }}'
+      ICON: '../appicon.png'
+      DESKTOP_FILE: '{{ "{{.APP_NAME}}" }}.desktop'
+      OUTPUT_DIR: '../../bin'
+
   generate:linux:dotdesktop:
```

Comment:
I think it would be worth passing in the subdirectory to this task rather than doing both. An example of this:

```
  build-app-prod-darwin:
    summary: Creates a production build of the application
    cmds:
      - task: pre-build
      - task: build-frontend
      - GOOS=darwin GOARCH={{ "{{.ARCH}}" }} go build -tags production -ldflags="-w -s" -o build/bin/{{ "{{.APP_NAME}}" }}
      - task: post-build
    env:
      CGO_CFLAGS: "-mmacosx-version-min=10.13"
      CGO_LDFLAGS: "-mmacosx-version-min=10.13"
      MACOSX_DEPLOYMENT_TARGET: "10.13"
    vars:
      ARCH: $GOARCH
```
Call with:
```
  package-darwin-arm64:
    summary: Packages a production build of the application into a `.app` bundle
    platform: darwin
    deps:
      - task: build-app-prod-darwin
        vars:
            ARCH: arm64
      - generate-icons
    cmds:
      - task: create-app-bundle
   ```

Reply:
I am not sure what do you mean. I am quite dumb to be honest. Do you want me to generate rpm separately?